ui/legacyHelpers.js

```diff
@@ -0,0 +1,21 @@
+// HELPERS
+import '../app/assets/javascripts/helpers/react_helpers.jsx';
+import '../app/assets/javascripts/helpers/graph_helpers.js';
+import '../app/assets/javascripts/helpers/feed_helpers.jsx';
+// (unordered)
+import '../app/assets/javascripts/helpers/from_pair.jsx';
+import '../app/assets/javascripts/helpers/graph_helpers.js';
+import '../app/assets/javascripts/helpers/mixpanel_utils.jsx';
+import '../app/assets/javascripts/helpers/profile_details_style.jsx';
+import '../app/assets/javascripts/helpers/prop_types.jsx';
+import '../app/assets/javascripts/helpers/sort_helpers.jsx';
+import * as Routes from '../app/assets/javascripts/helpers/Routes';
+import {colors, styles} from '../app/assets/javascripts/helpers/Theme';
+
+// Export these so legacy components can use this, and
+// new components can pull in components
+window.shared || (window.shared = {});
+window.shared.Routes = Routes;
+window.shared.colors = colors;
+window.shared.styles =styles;
```

Comment:
This adds the shim for helpers that are now ES6 modules, but where legacy components still expect to find them on `window.shared`.  This has to be in a separate file from `legacy.js`, since import statements are all hoisted and so these assignments run last regardless of where they are in the file.